Added wall api to app.json, fixed typo in kang and added help for /wall

Guru-25 opened the pull request.

Guru-25 committed:
Added config for wallpaper

Guru-25 committed:
Added wallpaper command in /help

Guru-25 committed:
Fixed typo in sticker kanging pack Name

TsunayoshiSawada commented on SaitamaRobot/modules/stickers.py:
```diff
@@ -376,15 +376,15 @@ def makepack_internal(
                 user.id,
                 packname,
-                f"{name}s kang pack" + extra_version,
+                f"{name}'s kang pack" + extra_version,
```
Gramatically makes more sense but as https://t.me/OnePunchDev/88178 said this would make Saitama ignore existing packs and create new packs, this causes high level of complains. 
So let's skip this one until we have a better solution for existing bots.

Guru-25 replied:
Yea. Right 👍

TsunayoshiSawada committed:
[revert] Fixed typo in sticker kanging pack Name

Reverting because this would make Saitama ignore existing packs and create new packs, this causes high level of complains.
